tools/integration_tests/util/operations/file_operations.go

```diff
@@ -141,3 +143,45 @@ func CloseFile(file *os.File) {
 		log.Printf("error in closing: %v", err)
 	}
 }
+
+func ReadFileSequentially(filePath string, numberOfBytes int64) (content []byte, err error) {
+	content = make([]byte, numberOfBytes)
+	chunk := make([]byte, NumberOfBytesReadFromFile)
+	var offset int64 = 0
+
+	file, err := os.OpenFile(filePath, os.O_RDWR|syscall.O_DIRECT, FilePermission_0600)
+	if err != nil {
+		log.Printf("Error in opening file:%v", err)
+	}
+
+	for err != io.EOF {
+		var numberOfBytes int
+
+		// Reading 200 bytes chunk sequentially from the file.
+		numberOfBytes, err = file.ReadAt(chunk, offset)
+		// If the file reaches the end, write the remaining content in the buffer and return.
+		if err == io.EOF {
+			for i := offset; i < offset+int64(numberOfBytes); i++ {
```

Comment:
can we use `append` or similar function to append two slices instead of using loop?